var/spack/repos/builtin/packages/net-snmp/package.py

```diff
@@ -10,9 +10,10 @@ class NetSnmp(AutotoolsPackage):
     """A SNMP application library, tools and daemon."""
 
     homepage = "http://www.net-snmp.org/"
-    url      = "https://sourceforge.net/projects/net-snmp/files/net-snmp/5.9/net-snmp-5.9.tar.gz"
+    url      = "https://sourceforge.net/projects/net-snmp/files/net-snmp/5.9.1/net-snmp-5.9.1.tar.gz/download"
 
     version('5.9', sha256='04303a66f85d6d8b16d3cc53bde50428877c82ab524e17591dfceaeb94df6071')
+    version('5.9.1', sha256='eb7fd4a44de6cddbffd9a92a85ad1309e5c1054fb9d5a7dd93079c8953f48c3f')
```

Comment:
We usually put newer versions at the top, can you fix that?

Reply:
Thanks! Done.